frontend/src/pages/main/home/client/scanPayEntry/ScanPayEntry.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {Box, Card, CardActionArea, CardContent, Typography} from "@mui/material";
+import CenterFocusWeakIcon from '@mui/icons-material/CenterFocusWeak';
+import "../../../../../common/styles/container.css";
+import "../../../../../common/styles/icons.css";
+import {ScanPayEntryProps} from "./props";
+import {Content} from "../../../../../common/constansts/content";
+
+const ScanPayEntry: React.FC<ScanPayEntryProps> = ({handlePaymentClick}) => {
+
+    return (
+        <Card className="card-payment">
+            <CardActionArea onClick={handlePaymentClick}>
+                <CardContent className="card-content">
+                    <Box className="home-components-info">
+                        <CenterFocusWeakIcon className="icon"/>
+                        <Typography variant="h5">
+                            {Content.SCAN_PAY}
+                        </Typography>
+                    </Box>
+                </CardContent>
+            </CardActionArea>
+        </Card>
```

Comment:
Try to make it same as in the design. Text and icons not so close to borders, text a bit smaller "h6". Maybe cards itself a bit bigger. 
Balance component looks good

Reply:
I found the problem. I viewed it on 27 inch monitor and it was small, but on the 14 inch it looks fine. Other words, your design is not quite responsive. Anyway, let's skip it and leave as it is, except h5 -> h6